spec/spec_helper.rb

```diff
@@ -48,7 +51,7 @@
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
 # The settings below are suggested to provide a good initial experience
-# with RSpec, but feel free to customize to your heart's content.
+# with RSpec, but feel free to customize to your heart"s content.
```

Comment:
Looks like some of the changes includes changing single quotes to double quotes which is fine but doesn't make sense to here. Super minor but thought you should know. Great stuff looking forward to this.